src/EditorFeatures/CSharpTest/Diagnostics/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -192,7 +192,7 @@ public void TestConflictingProperties()
         {
             Test(
 @"class Test : [|I1|] { int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ",
-@"using System ; class Test : I1 { int I1 . Prop { get { throw new NotImplementedException ( ) ; } set { throw new NotImplementedException ( ) ; } } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
+@"class Test : I1 { int I1 . Prop { get ; set ; } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
```

Comment:
Yes, we should add an option. @CyrusNajmabadi and I were just talking a few days ago about needing a "Prefer auto-properties" option to help control the "Use auto-property" diagnostic/fix.